ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/claimantresponse/CCDResponseRejection.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.cmc.ccd.domain.claimantresponse;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Value
+public class CCDResponseRejection extends CCDClaimantResponse {
+
+    private CCDYesNoOption freeMediationOption;
+
+    private String reason;
+
+    @Builder
+    @JsonCreator
+    public CCDResponseRejection(BigDecimal amountPaid,
```

Comment:
Do we need @jsoncreator, try without it.

Reply:
removed